src/core/sort.js

```diff
@@ -0,0 +1,12 @@
+/**
+ * @param {{priority: number}} a
+ * @param {{priority: number}} b
+ */
+
```

Comment:
I think you need to remove this blank line or VS Code can't associate the code with the JSDoc block.